2nd-gen/instrument-with-opentelemetry/functions/.env

```diff
@@ -0,0 +1 @@
+NODE_OPTIONS=--require=./tracing.js
```

Comment:
is this needed?

Reply:
yes - it's the only way I've found to load otel libraries before functions framework is loaded. Due to how monkeypatching works, otel sdk has to be the very first thing that runs before functions framework or user code is loaed.

Reply:
Interesting, I didn't know we could do this. Thanks for explaining!